python/requirements_ml_docker.txt

```diff
@@ -5,7 +5,7 @@ tblib
 
 # If you make changes to the torch versions, please also make the corresponding changes to `requirements_dl.txt`!
 -f https://download.pytorch.org/whl/torch_stable.html
-torch==1.9.0+cu111
+torch==1.11.0+cu111
```

Comment:
Unfortunately our dependencies for CI are a bit of a mess right now so upgrading won't work out of the box.

Instead can we do the following:
- Split out the fsdp test to a separate file (`test_torch_fsdp.py`)
- Add this file to `ray/train/tests/BUILD` with the following tags: `tags = ["team:ml", "exclusive", "gpu_only", "torch_1_11"]`
- Create a new test suite like this: https://github.com/ray-project/ray/blob/master/.buildkite/pipeline.gpu.large.yml#L1-L10, except with the following changes:
```
- label: ":tv: :steam_locomotive: Train GPU tests (PyTorch 1.11) "
  conditions: ["RAY_CI_TRAIN_AFFECTED"]
  commands:
    - cleanup() { if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then ./ci/build/upload_build_info.sh; fi }; trap cleanup EXIT
    - PYTHON=3.7 TRAIN_TESTING=1 ./ci/env/install-dependencies.sh
    # Because Python version changed, we need to re-install Ray here
    - rm -rf ./python/ray/thirdparty_files; rm -rf ./python/ray/pickle5_files; ./ci/ci.sh build
    - pip install -Ur torch==1.11.0+cu113 --extra-index-url https://download.pytorch.org/whl/cu113
    - ./ci/env/env_info.sh
    - bazel test --config=ci $(./ci/run/bazel_export_options) --build_tests_only --test_tag_filters=gpu,gpu_only,torch_1_11,-ray_air python/ray/train/...
```